providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/JsonBindingProvider.java

```diff
@@ -37,14 +43,52 @@
 @Priority(Priorities.USER-100)
 public class JsonBindingProvider extends AbstractJsonBindingProvider
         implements MessageBodyReader<Object>, MessageBodyWriter<Object> {
-
+   private static final boolean JSONB_DISABLED = Boolean.getBoolean("resteasy.jsonb.disable");
```

Comment:
Existing deployments do not have `resteasy.jsonb.disable` defined so the `JSONB_DISABLED` will be false.

Will those deployments work correctly without need for any changes in them ?

Reply:
There are some deployments that are going to be affected and need changes. That's also why Jim had to fix some tests in the PR here.
Note, we *can't* default to JSONB_DISABLED = true as that would break EE8 compliance.
So, IOW, there're still some scenarios in which JAX-RS 2.1/EE8 is forcing us to break backward compatibility (and this is also why I'm adding you as reviewer here)

Reply:
Shouldn't we introduce something like `resteasy.EE7.like.behavior` property?

This is not the first change which may require customers actions with deployments.
Customers do not want to touch deployments. And when we think about OpenShift, it's another reason why it should be good to have one system property for backward compatible behavior.

Reply:
This line reads system property "resteasy.jsonb.disable" if user needs a global settings for all resouces/deloyments.